### ISSUE TITLE
Allow automatically updated blacklists with comment filters

### ISSUE THREAD

**Issue by Dalethium** [yacy/yacy_search_server#446]
An ideal system would be being able to just slap a standard blacklist into a page and having it update daily, while ignoring clear comment lines such as `#` `//` and `;` 

https://blocklistproject.github.io/Lists/alt-version/abuse-nl.txt would be a good example of this.

Blacklists are an entire project of their own to maintain, offloading that work to someone else who already does it and having it auto update would be ideal.

**Comment by reger24** [comment]
Have a look at BlacklistImpExp_p.html -> "plain text file".
This does in princip exactly what you suggest (import a plain blacklist from text file) with last commit https://github.com/yacy/yacy_search_server/commit/11c4a1b45ca5cc3e3d8ecd853d81b5999c51f2c8 it excludes comment lines too.

I understand your idea for the "ideal system" grabs the file automatically & frequently in background.
Makes imho only sense if there would be a reliable and maintained source available used by at least some yacy user (don't see that/know of it), so the manual import is imho adequate.

The problem that the import page has difficulties to handle lists with 500.000 entries is a different topic, as YaCy would also have it's burden to handle huge blacklists.

**Comment by reger24** [comment]
fyi:  another way (workaround) to get what you suggest is.....  which at least would work with your example list above

1. create a new blacklist in yacy ( e.g.   "test")  -> this creates a file DATA/LISTS/test.black (which is a plain text file allowing comments starting with "#" as 1st char of line)
2. dump your desired blacklist text file with the name test.black in above directory (DATA/LISTS)
3. done - start YaCy and happily see 435321 new created entries in Blacklist_p.html

**Comment by Dalethium** [comment]
> I understand your idea for the "ideal system" grabs the file automatically & frequently in background. Makes imho only sense if there would be a reliable and maintained source available used by at least some yacy user (don't see that/know of it), so the manual import is imho adequate.

There are a lot of maintained blacklists out there, most of them designed for a wide variety of reasons but mostly to do with pihole and adblock -- The list I linked above is updated frequently, for example, and a yacy install would benefit from an automatic update of that URL.

> 3. done - start YaCy and happily see 435321 new created entries in Blacklist_p.html

Would this work while yacy is running? Maybe I could have an external program download the URL to the blacklist and just have yacy restart daily? That would be a good stop-gap solution right?

**Comment by reger24** [comment]
yes, that can be done while running. The update to the new downloaded list would be on restart,
or just viewing the list it in Blacklist_p.html

I guess for some time that would be the way to go.
